### DISCUSSION TITLE
最后raidrive报错

### DISCUSSION THREAD

**Discussion by wuwenchao1** [AlistGo/alist#3243] [Q&A]
[](url)
![Snipaste_2023-02-03_17-19-34](https://user-images.githubusercontent.com/122904772/216564029-6a137b05-2df9-4677-9cc6-f38d07918ffd.png)

**Comment by anwen-anyi** [comment]
https://alist.nn.ci/zh/guide/webdav.html 自行仔细查看文档！

**Comment by wuwenchao1** [comment]
Ok, thank you. I have installed the configuration web disk and mount, but
my computer is w10. How can the system set the script to boot on the
computer

安稳 ***@***.***> 于 2023年2月4日周六 13:34写道：
